test/bundleUtilConsts.js

```diff
@@ -0,0 +1,150 @@
+const URN_REPLACE_REFERENCES_ENTRIES = [
```

Comment:
A better name for this file would be `bundleUtilFixtures.js` since this is test fixture data.